core/amber/src/main/scala/edu/uci/ics/texera/web/resource/CollaborationResource.java

```diff
@@ -0,0 +1,42 @@
+package edu.uci.ics.texera.web.resource;
+
+import edu.uci.ics.texera.web.ServletAwareConfigurator;
+
+import javax.websocket.*;
+import javax.websocket.server.ServerEndpoint;
+import java.util.HashMap;
+
+
+@ServerEndpoint(
+        value = "/collaboration",
+        configurator = ServletAwareConfigurator.class
+)
+public class CollaborationResource {
+    public static HashMap<String, Session> websocketSessionMap = new HashMap<>();
+
+    @OnOpen
+    public void myOnOpen(final Session session) {
+        websocketSessionMap.put(session.getId(), session);
+    }
+
+    @OnMessage
+    public void myOnMsg(final Session session, String message) {
+
+        message = "{" + message.substring(1, message.length() - 1) + "}";
+
+        for(String key: websocketSessionMap.keySet()) {
```

Comment:
please give it a proper name rather than `key`. it's too general.

Reply:
Done.